### PR TITLE
build: upgrade library to GA

### LINKED ISSUE #120: promote library to GA
Package name: **FIXME**
Current release: **beta**
Proposed release: **GA**

## Instructions

Check the lists below, adding tests / documentation as required. Once all the "required" boxes are ticked, please create a release and close this issue.

## Required

- [ ] 28 days elapsed since last beta release with new API surface
- [x] Server API is GA
- [x] Package API is stable, and we can commit to backward compatibility
- [x] All dependencies are GA

## Optional

- [ ] Most common / important scenarios have descriptive samples
- [ ] Public manual methods have at least one usage sample each (excluding overloads)
- [ ] Per-API README includes a full description of the API
- [ ] Per-API README contains at least one “getting started” sample using the most common API scenario
- [ ] Manual code has been reviewed by API producer
- [ ] Manual code has been reviewed by a DPE responsible for samples
- [ ] 'Client Libraries' page is added to the product documentation in 'APIs & Reference' section of the product's documentation on Cloud Site

### PR DESCRIPTION
**Pull request by sofisl**
Fixes #120

### CODE REVIEW

File: .repo-metadata.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "product_documentation": "https://cloud.google.com/secret-manager/docs",
   "client_documentation": "https://googleapis.dev/nodejs/secretmanager/latest/index.html",
   "issue_tracker": "https://issuetracker.google.com/issues/new?component=784854",
-  "release_level": "beta",
+  "release_level": "GA",
```
Comment (bcoe) [inline .repo-metadata.json:7]: silly caveat, but I think we look for lowercase `ga`.

**Comment by sofisl** [comment]
@bcoe @JustinBeckwith - Sorry it took so long, had some problems installing synthtool. All good now!

**Comment by JustinBeckwith** [comment]
Thanks!  Looks like we have a merge conflict though :)

**Comment by bcoe** [comment]
thanks @sofisl \o/

**Pull request merged**